doc/source/visualizing/plots.rst

```diff
@@ -706,29 +706,98 @@ two element tuples.
+will invert the plot's y-axis:
 
 .. python-script::
 
    import yt
 
    ds = yt.load("IsolatedGalaxy/galaxy0030/galaxy0030")
-   # slicing with non right-handed coordinates
-   slc = yt.SlicePlot(ds, "x", ("gas", "velocity_x"), right_handed=False)
-   slc.annotate_title("Not Right Handed")
+   # slicing with standard view (right-handed)
+   slc = yt.SlicePlot(ds, "x", ("gas", "velocity_x"))
+   slc.annotate_title("Standard Horizontal (Right Handed)")
+   slc.save("Standard.png")
+
+   # flip the horizontal axis (not right handed)
+   slc.flip_horizontal()
+   slc.annotate_title("Horizontal Flipped (Not Right Handed)")
    slc.save("NotRightHanded.png")
 
-   # switching to right-handed coordinates
-   slc.toggle_right_handed()
-   slc.annotate_title("Right Handed")
-   slc.save("Standard.png")
+   # flip the vertical axis
+   slc = yt.SlicePlot(ds, "x", ("gas", "velocity_x"), flip_vertical=True)
+   slc.annotate_title("Flipped vertical")
+   slc.save("FlippedVertical.png")
+
+In addition to inverting the direction of each axis,
+:meth:`~yt.visualization.plot_window.AxisAlignedSlicePlot.swap_axes` will exchange
+the plot's vertical and horizontal axes:
+
+.. python-script::
+
+   import yt
+
+   ds = yt.load("IsolatedGalaxy/galaxy0030/galaxy0030")
+   # slicing with non right-handed coordinates
+   slc = yt.SlicePlot(ds, "x", ("gas", "velocity_x"), swap_axes=True)
+   slc.annotate_title("Swapped axes")
+   slc.save("SwappedAxes.png")
+
+   # toggle swap_axes (return to standard view)
+   slc.swap_axes()
+   slc.annotate_title("Standard Axes")
+   slc.save("StandardAxes.png")
+
+Note that when using ``swap_axes``, any plot modifications relating to limits,
+image width or resolution should still be supplied in reference to the standard
+(unswapped) orientation rather than the swapped view.
+
+It's worth mentioning that these three methods can be used in combination to
+rotate the view:
+
+.. python-script::
+
+   import yt
+
+   ds = yt.load("IsolatedGalaxy/galaxy0030/galaxy0030")
+   # initial view
+   slc = yt.SlicePlot(ds, "x", ("gas", "velocity_x"))
+   slc.save("InitialOrientation.png")
+   slc.annotate_title("Initial View")
+
+   # vertical flip + swap = rotate 90 degree rotation (clockwise)
+   slc.flip_vertical()
+   slc.swap_axes()
+   slc.annotate_title("90 Degree Clockwise Rotation")
+   slc.save("SwappedAxes90CW.png")
+
+   # vertical flip + horizontal flip = rotate 180 degree rotation
+   slc = yt.SlicePlot(ds, "x", ("gas", "velocity_x"))
+   slc.flip_horizontal()
+   slc.flip_vertical()
+   slc.annotate_title("180 Degree Rotation")
+   slc.save("FlipAxes180.png")
+
+   # horizontal flip + swap = rotate 90 degree rotation (counter clockwise)
+   slc = yt.SlicePlot(ds, "x", ("gas", "velocity_x"))
+   slc.flip_horizontal()
+   slc.swap_axes()
+   slc.annotate_title("90 Degree Counter Clockwise Rotation")
+   slc.save("SwappedAxes90CCW.png")
```

Comment:
|  These examples worked because I reset the figure's state every time, but here you're applying all operations in a single script without regenerating the initial figure, so I don't think it'll work. 

Is this a docs-specific thing? Seemed to work fine while testing the code itself (but I didn't re-build the docs).

Reply:
Oh sorry, reading this again I see that you've actually already done what I suggested... sorry about that, carry on!